docs/user/filesystem.md

```diff
@@ -944,7 +944,8 @@ the unified namespace integration.
 
 libpil4dfs is similar to libioil, but it intercepts not only read/write, but also
 metadata related functions. This provides similar performance as using native DFS
-with POSIX interface.
+with POSIX interface. libpil4dfs can be used in conjunction with dfuse or without
+a dfuse mountpoint.
```

Comment:
there are several limitation when running without a dfuse mountpoint i believe.
so i think it would be advisable to use a dfuse mountpoint for now and say the other path is non-production ready for now.